ibis/expr/api.py

```diff
@@ -3988,6 +3988,23 @@ def _table_union(left, right, distinct=False):
     return op.to_expr()
 
 
+def _table_intersect(left, right):
+    """
+    Form the table set intersect of two table expressions having identical
+    schemas.
```

Comment:
I think it would be nice to briefly explain what is the intersect of two tables.

Reply:
Ok I can definitely add that